msg_hash.h

```diff
@@ -1761,6 +1761,7 @@ enum msg_hash_enums
    MENU_LABEL(CONTENT_INFO_DATABASE),
    MENU_LABEL(CONTENT_INFO_RUNTIME),
    MENU_LABEL(CONTENT_INFO_LAST_PLAYED),
+   MENU_LABEL(CONTENT_INFO_RETROACHIEVEMENTS_HASH),
```

Comment:
maybe `CONTENT_INFO_CHEEVOS_HASH` for consistency sake?

Reply:
There isn't really any cohesive consistency between the existing labels. _Most_ are CHEEVOS, but there's a couple that use "RETRO_ACHIEVEMENTS". So I'm not really consistent with either. Will change to CHEEVOS.